### PR TITLE
Adapt PPC to use workload hints instead of setting additional kernel arguments

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Now power consumption modes will work together with workload hints:

1. low-latency -> enable real-time workload hint
2. ultra-low-latency -> enable realtime and highPowerConsumtion hints

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: cmd/performance-profile-creator/cmd/root.go [hunk 1]
```diff
@@ -44,22 +44,59 @@ const (
...
+	nodeSelector              *metav1.LabelSelector
+	mcpSelector               map[string]string
+	performanceProfileName    string
+	topologyPolicy            string
+	rtKernel                  bool
+	additionalKernelArgs      []string
+	userLevelNetworking       *bool
+	disableHT                 bool
+	realtimeHint              *bool
+	highPowerConsumptionHints *bool
```
Comment (yanirq) [inline cmd/performance-profile-creator/cmd/root.go:99]: s/Hints/Hint   (or is this suppose to be in plural?)

Comment (cynepco3hahue) [reply cmd/performance-profile-creator/cmd/root.go:99]: done

**Comment by yanirq** [comment]
After a first round the PR looks ok.  I will need a second look and maybe others will want to have a review.
We do have d/s , but maybe we should have an u/s equivalent as well with these changes ? 

In addition, if you feel the PR is ready then please squash the commits

**Comment by cynepco3hahue** [comment]
> After a first round the PR looks ok. I will need a second look and maybe others will want to have a review. We do have d/s , but maybe we should have an u/s equivalent as well with these changes ?
> 
> In addition, if you feel the PR is ready then please squash the commits

@yanirq The PR rebased.

**Comment by yanirq** [comment]
/lgtm
@cynepco3hahue  @jlojosnegros  lets make sure that there is a followup PR with docs

**Pull request merged**